Makefile.am

```diff
@@ -294,18 +297,63 @@ libpsmx_fi_la_LDFLAGS = \
 libpsmx_fi_la_LIBADD = $(linkback) $(psm_LIBS)
 libpsmx_fi_la_DEPENDENCIES = $(linkback)
 else !HAVE_PSM_DL
-src_libfabric_la_SOURCES += $(_psm_files)
-src_libfabric_la_CPPFLAGS += $(psm_CPPFLAGS)
-src_libfabric_la_LDFLAGS += $(psm_LDFLAGS)
-src_libfabric_la_LIBADD += $(psm_LIBS)
+noinst_LTLIBRARIES += libpsmx.la
```

Comment:
Just curious: is there a reason to make this (and `libpsmx2.la`) separate convenience LT libraries (vs. just including the sources directly in the `src_libfabric_la_SOURCES`)?